src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
 
 public class Main {
+    private static final Logger logger = (Logger) LogManager.getLogger(Main.class);
```

Comment:
Casting `LogManager.getLogger(Main.class)` to `Logger` is unnecessary. You can directly assign it to `logger` without casting.